api/src/main/java/org/pmiops/workbench/auth/UserAuthentication.java

```diff
@@ -9,14 +9,21 @@
 
 public class UserAuthentication implements Authentication {
 
+  public enum UserType {
+    RESEARCHER,
```

Comment:
So this could get slightly confusing, because the researcher's pet SA can also trigger this user type. Perhaps add a comment here and/or in the WorkbenchConfig.

Reply:
Added comment here and in WorkbenchConfig.